x-pack/plugins/observability/server/index.ts

```diff
@@ -22,19 +22,26 @@ export { getInspectResponse } from '../common/utils/get_inspect_response';
 
 export * from './types';
 
+const configSchema = schema.object({
+  annotations: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
+    index: schema.string({ defaultValue: 'observability-annotations' }),
+  }),
+  unsafe: schema.object({
+    slo: schema.object({
+      enabled: schema.boolean({ defaultValue: false }),
+    }),
+  }),
+});
+
 export const config: PluginConfigDescriptor = {
   exposeToBrowser: {
-    unsafe: true,
+    unsafe: false,
```

Comment:
As we are introducing another feature flag `alertDetail` which is exposed to the client side and it's under `unsafe` too.